include/sound/hdmi-codec.h

```diff
@@ -34,11 +34,6 @@ struct hdmi_codec_daifmt {
 	unsigned int frame_clk_inv:1;
 	unsigned int bit_clk_master:1;
 	unsigned int frame_clk_master:1;
-	/* bit_fmt could be standard PCM format or
-	 * IEC958 encoded format. ALSA IEC958 plugin will pass
-	 * IEC958_SUBFRAME format to the underneath driver.
-	 */
-	snd_pcm_format_t bit_fmt;
 };
```

Comment:
Please don't do that, I'm using it in currently out of tree changes for DW-HDMI audio, so I was pretty happy to see it's added.